### PR TITLE
test: add unit tests for process_submissions helpers

### LINKED ISSUE #1601: Create unit tests for ingester's process submission file
process_submissions.py defines the functions that receive submission data (dict) and creates the instances of each model in that submission.
Detached from #1569 so that we can be more granular with the tests

### PR DESCRIPTION
**Pull request by joao-faria-dev**
## Related issue:
- Closes #1601

## Description 
Add test coverage for the helpers functions on `process_submissions.py`

### CODE REVIEW

File: backend/kernelCI_app/tests/unitTests/helpers/process_submissions_test.py [hunk 1]
```diff
@@ -0,0 +1,559 @@
...
+        self.assertEqual(result["issues"][0], mock_issue_1)
+        self.assertEqual(result["issues"][1], mock_issue_3)
+        self.assertEqual(mock_make_issue.call_count, 3)
+
+
+class TestInsertItems(SimpleTestCase):
+    @patch("kernelCI_app.management.commands.helpers.process_submissions.logger")
+    @patch(
+        "kernelCI_app.management.commands.helpers.process_submissions.make_issue_instance"
+    )
```
Comment (gustavobtflores) [inline backend/kernelCI_app/tests/unitTests/helpers/process_submissions_test.py]: we could add a test case with an empty items list in this class, just in case

**Comment by joao-faria-dev** [comment]
Just rebased @gustavobtflores
